packages/host/app/resources/search.ts

```diff
@@ -85,11 +195,32 @@ export function getSearchResults(
   parent: object,
   query: Query,
   realms?: string[],
+  opts?: {
+    // A new search is triggered whenever the index updates. Consumers of this
+    // function that render their cards using {{#each}} should make sure to use the
+    // "key" field: {{#each #this.results.instances key="id" as |instance|}} in
```

Comment:
Is this actually needed? Aren't we keeping the instances the same on updates via updateFromSerialized?

Reply:
ah, good point. I just copy pasted this from the previous implementation, but you are right.